components/layout_2020/display_list/mod.rs

```diff
@@ -175,6 +175,7 @@ impl DisplayList {
         context: &LayoutContext,
         fragment_tree: &FragmentTree,
         root_stacking_context: &StackingContext,
+        highlighted_dom_node: Option<OpaqueNode>,
```

Comment:
Hrm. Instead of the extra argument could you move the `highlighted_dom_node` to `LayoutContext` (as it is a property of the entire layout and not just display list building, in a sense) and add a bit of rustdoc contextualizing it? Thanks!